clang/include/clang/Driver/Options.td

```diff
@@ -4732,6 +4732,9 @@ def fno_sycl_device_lib_EQ : CommaJoined<["-"], "fno-sycl-device-lib=">, Group<s
   Values<"libc, libm-fp32, libm-fp64, all">, HelpText<"Control exclusion of "
   "device libraries from device binary linkage. Valid arguments "
   "are libc, libm-fp32, libm-fp64, all">;
+def fsycl_fp32_prec_sqrt : Flag<["-"], "fsycl-fp32-prec-sqrt">, Group<sycl_Group>, Flags<[CC1Option]>,
+  HelpText<"SYCL only. Specify that single precision floating-point sqrt used in the program source are correctly rounded.">,
```

Comment:
is correctly rounded*

Also do we need to say "in program source"

Is - `"SYCL only. Specify that single precision floating-point sqrt is correctly rounded."` suficient?

Reply:
Updated as suggested